src/llmq/quorums_signing.cpp

```diff
@@ -39,6 +39,50 @@ UniValue CRecoveredSig::ToJson() const
 CRecoveredSigsDb::CRecoveredSigsDb(CDBWrapper& _db) :
     db(_db)
 {
+    if (Params().NetworkIDString() == CBaseChainParams::TESTNET) {
```

Comment:
Is there a reason why this only affects testnet?

Reply:
Reason is that only testnet currently contains any of these values in the DB. Mainnet doesn't have these keys/values, even if you already run v14 on mainnet. I'd like to avoid having this `rs_upgraded` key to appear on mainnet. Also, I'd like to remove this upgrade code ASAP.

Reply:
Sounds good, that's what I presumed